pkg/asset/agent/configimage/configiso.go

```diff
@@ -65,6 +65,14 @@ func (a *ConfigImage) Generate(dependencies asset.Parents) error {
 		}
 	}
 
+	// Add additional file to allow for override of setting the core password
+	if _, ok := os.LookupEnv("OPENSHIFT_APPLIANCE_OVERRIDE_PASSWORD_SET"); ok {
```

Comment:
This solution is problematic... IIUC, it means that we'll have to instruct all appliance users to manually set this env var - not a great user experience.
Instead, can you just add the file to [load-config-iso@.service.template](https://github.com/openshift/installer/blob/8cd8f79a20b326fb4bcb25f65580323b638ee326/data/data/agent/systemd/units/load-config-iso@.service.template#L6-L7)?
I.e.:
```
EnvironmentFile=/etc/assisted/appliance-override-password-set
```
Then, the appliance will just ensure to create this file on bootstrap ignition.

Reply:
OK, changed to just check the file directly that could be added in appliance flow

Reply:
Yeah, we don't use environment variables as an API to control behaviour of the installer.
Looking for a file in the image creates an API between the appliance builder and the installer that we have to keep stable, but this is still preferable.